src/Prometheus/Storage/APC.php

```diff
@@ -11,21 +11,26 @@
 
 class APC implements Adapter
 {
-    const PROMETHEUS_PREFIX = 'prom';
+    /** @var string Prefix to use for APC keys. */
+    private $prometheusPrefix;
 
     /**
      * APC constructor.
      *
+     * @param string $prometheusPrefix Prefix for APCu keys (defaults to 'prom').
+     *
      * @throws StorageException
      */
-    public function __construct()
+    public function __construct(string $prometheusPrefix = 'prom')
```

Comment:
I would prefer to reuse the const here:
```suggestion
    public function __construct(string $prometheusPrefix = self::PROMETHEUS_PREFIX)
```